api/src/lib.rs

```diff
@@ -1540,20 +1606,50 @@ where
     }
 
     {
-        // Verification tokens
-        let mut verify_tokens =
+        // Legacy verification endpoint. Remove for API v3
+        let mut verify_tokens_legacy =
             tide::with_state(State::new(verify_db.clone(), token_db.clone()));
 
-        verify_tokens
+        verify_tokens_legacy
             .at(":id")
             .with(ensure_vault_id)
-            .with(cors)
+            .with(signed_pow_auth)
```

Comment:
@ceberly we need to adjust this to make sure the PUT is not behind auth, right? (re: the convo on 93)

Reply:
see [78b7bc8](https://github.com/withuno/identity/pull/92/commits/78b7bc8457bb26bd5e3f0ad2f62ad723019464f2)